src/data/pnat/divisors.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2019 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Yury Kudryashov
+-/
+import data.fintype data.pnat.basic tactic.alias
+
+/-!
+# Divisors of a positive natural number
+
+Given a number `n : ℕ+`, `divisors n` is the set of all positive
```

Comment:
It's defined as a subtype of `pnat`, not a set. Could you add a bit more to the header about implementation details? All defs in this file need doc strings.